articles/configure-your-site-navigation.mdx

```diff
@@ -2,103 +2,106 @@
+### Collections
 
-The list of panels in your site is configurable in your site and build information. CloudCannon gives you plenty of opportunities to define the order, type, and information of items displayed in these panels. Learn more about how to change the items in your site navigation.
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/editing/navigation/2024-10/CloudCannon-Documentation-Site-Navigation-Collections.png" alt="A screenshot of the Site Navigation shows links to the Collections on this Site, organized into groups." title="The Site Navigation" type="ui-snippet"/>
 
-**Collections section**
+In the middle of the scrollable section in your *Site Navigation* are the links for your Collections.
 
-You can define, add, rename, and group collections in your configuration settings.
+* **Collections** — Groups of related content, such as a folder of blog articles, Staff Member profiles, or product pages. Clicking on a Collection will open the Collection browser.
 
-* **Pages** — By default, all of your site's pages will appear here.
-* **Blogging** — Default with a blog included.
-  * **Posts** — A library of your published blog articles.
-  * **Drafts** — All of your unpublished blog articles.
-* **Collections/data** — Set of custom content defined by your setup or data file. Examples of collections might include products, staff members, or articles.
+By default, CloudCannon will sort your Collections in the following order: the *Pages* Collection, all output Collections in alphabetical order, then all non-output Collections in alphabetical order.
 
-<comp.DocsImage path="ye_olde_images/editing-navigation-site-navigation-collections@2.png" alt="Snippet of the collections section in the Site Navigation." title="" type="screenshot-snippet"/>
+You can configure the order of your Collections using the [Collection Groups](/documentation/articles/configure-your-site-navigation/#collection_groups) option.
 
-CloudCannon groups your collections, pages, and data in your site's navigation. You can reorder, show/hide, and regroup sections with `collection_groups` in your global configuration file.
+### Developer Tools
 
-<comp.MultiCodeBlock language="YAML" translate_into={["json"]} source="cloudcannon.config.yml">
-```
-collection_groups:
-  - heading: Blogging
-    collections:
-      - posts
-      - drafts
-  - heading: Other
-    collections:
-      - pages
-      - staff
-      - features
-```
-</comp.MultiCodeBlock>
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/editing/navigation/2024-10/CloudCannon-Documentation-Site-Navigation-Developer-Tools.png" alt="A screenshot of the Site Navigation shows links to the Status, Files, and Site Settings pages." title="The Site Navigation" type="ui-snippet"/>
 
-## Configure dashboard actions
+At the bottom of the scrollable section in your *Site Navigation* are your Developer tools. These are important for managing the Site itself, rather than it's content.
```

Comment:
```suggestion
At the bottom of the scrollable section in your *Site Navigation* are your Developer tools. These are important for managing the Site itself, rather than its content.
```